network.go

```diff
@@ -58,8 +61,44 @@ const (
 	// when requested by the runtime
 	// This will be used for vethtap, macvtap, ipvtap
 	ModelEnlightened
+
+	// InvalidNetInterworkingModel is the last item to check valid values by IsValid()
+	InvalidNetInterworkingModel
 )
 
+//IsValid checks if a model is valid
+func (n NetInterworkingModel) IsValid() bool {
```

Comment:
@sboeuf will be used by vc caller, in this case the runtime to allow choose the default model. https://github.com/clearcontainers/runtime/pull/887/files#diff-b4bda758a2aef091432646c354b4dc59R397

It could set it using any of the NetXConnect*Model values but as this is going to be user defined  (From configuration.toml) I opted to set the model by using an string representation.